src/test/java/teammates/test/cases/browsertests/FeedbackRankQuestionUiTest.java

```diff
@@ -359,6 +400,60 @@ public void testEditQuestionAction() throws Exception {
         feedbackEditPage.clickSaveExistingQuestionButton(2);
         feedbackEditPage.verifyStatus(Const.StatusMessages.FEEDBACK_QUESTION_EDITED);
         assertTrue(feedbackEditPage.isRankDuplicatesAllowedChecked(2));
+
+        ______TS("Rank options: test min/max restrictions");
+
+        int qNum = 1;
+        feedbackEditPage.clickEditQuestionButton(qNum);
+
+        // checking max options to be ranked restriction
```

Comment:
This comment is not easy to understand. A comment should be meaningful to an 'outsider', not just the dev :-)